### PR TITLE
Updated to API 7.0

### PR DESCRIPTION
**Pull request by 0dd-b1t**
Please double-check the CloudStorage interface.

### CODE REVIEW

File: dist/index.d.ts [hunk 1]
```diff
@@ -269,6 +295,98 @@ export declare namespace TelegramWebApps {
...
+         * A method that stores a value in the cloud storage using the specified key. 
+         * The key should contain 1-128 characters, only A-Z, a-z, 0-9, _ and - are allowed. 
+         * The value should contain 0-4096 characters. You can store up to 1024 keys in the cloud storage. 
+         * 
+         * If an optional callback parameter was passed, the callback function will be called. 
+         * 
+         * In case of an error, the first argument will contain the error. 
+         * In case of success, the first argument will be null and the second argument will be a boolean indicating whether the value was stored.
+         */
+        setItem(key: string, value: string, callback?: (result:any, confirm: boolean) => void): void;
```
Comment (matvejs16) [inline dist/index.d.ts]: This and all other methods below should look like this:
(I'm not sure about "Error" type in error argument, need to be checked)
```typescript
setItem(key: string, value: string, callback?: (error: Error | null, confirm: boolean) => void): CloudStorage;
```

```All these methods return the HapticFeedback object so they can be chained.```

File: dist/index.d.ts [hunk 2]
```diff
@@ -269,6 +295,98 @@ export declare namespace TelegramWebApps {
...
+         * 	Shows whether the context menu item is visible. Set to false by default.
+         */
+        isVisible: boolean;
+        /**
+         * Bot API 7.0+ 
+         * A method that sets the press event handler for the Settings item in the context menu. 
+         * 
+         * An alias for Telegram.WebApp.onEvent('settingsButtonClicked', callback)
+         */
+        onClick(callback: () => void): void;
```
Comment (matvejs16) [inline dist/index.d.ts]: This and all other methods below should return SettingsButton interface.
```All these methods return the SettingsButton object so they can be chained.```

**Review by matvejs16 — CHANGES_REQUESTED** [review]
Need some changes (made comments where it's needed)

**Comment by 0dd-b1t** [comment]
Should be done now

File: dist/index.d.ts
Comment (matvejs16) [inline dist/index.d.ts:1]: LGTM

**Review by matvejs16 — APPROVED** [review]

**Pull request merged**